src/main/java/de/retest/recheck/ignore/SearchFilterFiles.java

```diff
@@ -23,4 +37,19 @@ private SearchFilterFiles() {}
 				.collect( Collectors.toList() ); //
 	}
 
+	public static List<File> getProjectFilterFiles() {
+		if ( !resolveFilterPath().toFile().exists() ) {
+			ProjectConfiguration.getInstance().ensureProjectConfigurationInitialized();
```

Comment:
Can't we just assume, that the project is initialized, since a get() should not execute other stuff. I would suggest, just returning an empty list. 

It should be called at startup anyways. @martin-v